sigs.yaml

```diff
@@ -1330,12 +1330,16 @@ sigs:
   label: instrumentation
   leadership:
     chairs:
+    - github: ehashman
+      name: Elana Hashman
+      company: Red Hat
+    - github: logicalhan
+      name: Han Kang
+      company: Google
+    tech_leads:
     - github: brancz
       name: Frederic Branczyk
       company: Red Hat
```

Comment:
Let's add @piosz as emeritus_lead.

```
emeritus_leads:
    - github: piosz
      name: Piotr Szczesniak
      company: Google
```

Reply:
Did not include the company because other emeritus_leads in here didn't include that.